views/confirm_delete.templ

```diff
@@ -23,7 +23,9 @@ templ ConfirmDelete(args ConfirmDeleteArgs) {
 					</h2>
 				</div>
 				<div class="modal-body">
-					<p>{ args.Question }</p>
+					<p>
+						@templ.Raw(args.Question)
+					</p>
```

Comment:
Heads up: had to wrap this with `@templ.Raw` in order to show the file name in bold:

<img width="523" alt="image" src="https://github.com/ugent-library/biblio-backoffice/assets/599538/e7ec35b5-8cb1-40b7-8855-988dbf3398ee">